RecoEcal/EgammaCoreTools/test/testEcalClusterLazyTools.cc

```diff
@@ -102,9 +102,9 @@ void testEcalClusterLazyTools::analyze(const edm::Event& ev, const edm::EventSet
     std::cout << std::endl;
     std::vector<float> vLat = lazyTools.lat(clus);
     std::cout << "lat...................... " << vLat[0] << " " << vLat[1] << " " << vLat[2] << std::endl;
-    std::vector<float> vCov = lazyTools.covariances(clus);
+    std::array<float,3> vCov = lazyTools.covariances(clus);
```

Comment:
here and everywhere else below one could use ```auto``` instead of the explicit type (btw if auto was used in first place no change was required!)
This is somehow stylistic (I prefer auto)  so I leave to reco-conv. too say the final word

Reply:
```C++
const auto& vCov = lazyTools.covariances(clus);
```
seems like a good choice?